src/utils.py

```diff
@@ -0,0 +1,12 @@
+def calculate_sum(lst):
+   """
+   This function sums nested list.
+   parameters
+   -----------
+   list 
+   return
+   -------
+   array of values
```

Comment:
```suggestion
   
   Parameters
   -----------
   lst: List
       A list of values to be summed.
       
   Returns
   --------
   total: int
         sum of all input values
```

Please apply the similar changes as suggested below to all other functions.

Reply:
Fixed